File: solutions/cpp/raindrops/1/raindrops.cpp
#include "raindrops.h"

namespace raindrops
{

    std::string convert(unsigned int number)
    {
        std::string raindrop_sounds = "";

        if (number % 3 == 0)
        {
            raindrop_sounds.append("Pling");
        }

        if (number % 5 == 0)
        {
            raindrop_sounds.append("Plang");
        }

        if (number % 7 == 0)
        {
            raindrop_sounds.append("Plong");
        }

        if (raindrop_sounds.length() == 0)
        {
            raindrop_sounds = std::to_string(number);
        }

        return raindrop_sounds;
    }

} // namespace raindrops

File: solutions/php/raindrops/1/Raindrops.php
<?php

declare(strict_types=1);

function raindrops(int $number): string
{
    $result = "";

    if ($number % 3 == 0) $result = $result . 'Pling';
    if ($number % 5 == 0) $result = $result . 'Plang';
    if ($number % 7 == 0) $result = $result . 'Plong';
    if (strlen($result) == 0) $result = "$number";

    return $result;
}
